azerothcore/azerothcore-wotlk
# Fix(DB/SAI) update quest 6002

## Changes Proposed:

This PR proposes changes to:

## Issues Addressed:

- Closes 

## SOURCE:

The changes have been validated through:

## Tests Performed:

This PR has been:


## How to Test the Changes:



1. .go cre 26661
2. .q add 6002
3. see that the text field while the mission is in progress is no longer blank. Quest progression text on NPC was missing
4. Lunaclaw now targets the player without them having to get close, thus preventing the creature from disappearing if the player summons it and remains still.
5. Additionally, the IDs of the mission areas and points of interest have also been corrected.

## reference: 
- [x] https://www.wowhead.com/wotlk/quest=6002/body-and-heart
- [x] https://wowpedia.fandom.com/wiki/Body_and_Heart#Progress
<img width="1265" alt="image" src="https://github.com/azerothcore/azerothcore-wotlk/assets/122452427/3d5cac97-94b4-45b2-acf2-d02810a64380">
<img width="389" alt="image" src="https://github.com/azerothcore/azerothcore-wotlk/assets/122452427/1628c5ae-9193-4a6a-9709-dbcc95fb0e96">

## VIDEO BLIZZARD
- https://github.com/azerothcore/azerothcore-wotlk/assets/122452427/19e19a02-67bd-4419-a1d8-5ea26faf1c94

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
